README.md

```diff
@@ -542,6 +542,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Modern Java - A Guide to Java 8](https://github.com/winterbe/java8-tutorial) - Popular Java 8 guide.
 * [Modernizer](https://github.com/andrewgaul/modernizer-maven-plugin) - Detect uses of legacy Java APIs.
 * [OpenRefine](http://openrefine.org/) - Tool for working with messy data: cleaning, transforming, extending it with web services and linking it to databases.
+* [Polyglot for Maven](https://github.com/takari/polyglot-maven/) - Set of extensions for Maven 3.3.1+ that allows the POM model to be written in dialects other than XML.
```

Comment:
Please create another PR and don't do Add/Delete in a single one since deletes are much more likely to be open for discussion.